static/js/timerender.js

```diff
@@ -179,13 +179,26 @@ export function render_date(time, time_above, today) {
 }
 
 // Renders the timestamp returned by the <time:> Markdown syntax.
-export function render_markdown_timestamp(time, text) {
+export function render_markdown_timestamp(time) {
     const hourformat = page_params.twenty_four_hour_time ? "HH:mm" : "h:mm a";
     const timestring = format(time, "E, MMM d yyyy, " + hourformat);
-    const titlestring = "This time is in your timezone. Original text was '" + text + "'.";
+    const timezone = new Intl.DateTimeFormat(undefined, {timeZoneName: "short"})
+        .formatToParts(time)
+        .find(({type}) => type === "timeZoneName").value;
+
+    const tz_offset = -time.getTimezoneOffset() / 60;
+    const tz_offset_str =
+        timezone === "UTC" ? "" : " (UTC" + (tz_offset >= 0 ? "+" : "-") + tz_offset + ")";
+
+    const tooltipContent =
+        "Time zone: " +
+        (/GMT[+-][\d:]*/.test(timezone) ? "" : timezone) + // Avoid showing both GMT & UTC
+        tz_offset_str +
+        "<br/>Each user sees this time in their own time zone.";
```

Comment:
After playing with this for a bit, I think we want:
* "Everyone sees this in their own timezone." as the first line
* "Your time zone: {timezone}" as the second line.

We need to tag these two lines for translation.

Reply:
If this is HTML, its name needs to be marked `_html` and its interpolated strings need to be escaped.

Reply:
Extracted that to a template `static/templates/markdown_time_tooltip.hbs`.

Reply:
![image](https://user-images.githubusercontent.com/44665669/124315236-45a34180-db91-11eb-8ca5-67b58f175a82.png)
![image](https://user-images.githubusercontent.com/44665669/124315258-52c03080-db91-11eb-83d0-c439cf1d8304.png)